cmd/gen-manifests/main.go

```diff
@@ -214,22 +214,20 @@ func makeManifestJob(
 			return
 		}
 
-		var packageSpecs map[string][]rpmmd.PackageSpec
-		var repoConfigs map[string][]rpmmd.RepoConfig
+		var depsolvedSets map[string]dnfjson.DepsolveResult
 		if content["packages"] {
-			packageSpecs, repoConfigs, err = depsolve(cacheDir, manifest.GetPackageSetChains(), distribution, archName)
+			depsolvedSets, err = depsolve(cacheDir, manifest.GetPackageSetChains(), distribution, archName)
 			if err != nil {
 				err = fmt.Errorf("[%s] depsolve failed: %s", filename, err.Error())
 				return
 			}
-			if packageSpecs == nil {
+			if depsolvedSets == nil {
```

Comment:
I'm wondering if this check is now actually accurate. Since the `depsolvedSets` may not be `nil` and have repos set, but its `.Packages` may be empty...

Reply:
Yeah, I tweaked the check